.github/workflows/ci.yml

```diff
@@ -83,6 +84,11 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Set up NodeJS
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node }}
```

Comment:
We need to also set the node version like here https://github.com/o1-labs/proof-systems/blob/befc0ae73fbd9bf6f16d424e46e0726a724791cb/.github/workflows/ci.yml#L81
I believe this step is needed for new runners that don't provide NodeJS runtime by default (any new custom runner).

Reply:
Addressed, re-review?